tests/cpi-returns/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I _think_ the way it works / what worked for me is that you dont have to specify the dependencies here. The `package.json` file in the `tests` dir already has them and this folder can pull them as long as its defined as a npm workspace member in the `package.json` in `tests`

Reply:
Yea that is right